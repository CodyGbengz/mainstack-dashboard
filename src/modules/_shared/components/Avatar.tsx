import styled from 'styled-components';

import userimage from '../../../assets/userimage.png';

export const Avatar = () => {
  return (
    <AvatarContainer>
      <img
        src={userimage}
        alt='user avatar'
      />
    </AvatarContainer>
  );
};

const AvatarContainer = styled.div`
  border-radius: 2rem;
  overflow: hidden;
  height: 2rem;
  width: 2rem;
`;
