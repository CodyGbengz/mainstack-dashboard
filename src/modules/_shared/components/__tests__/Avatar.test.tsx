import { render } from '@testing-library/react';
import { Avatar } from '../Avatar';

describe('COMPONENT TEST: Avatar', () => {
  const setup = () => render(<Avatar />);

  test('Should match snapshot', () => {
    const { container } = setup();
    expect(container.parentElement).toMatchSnapshot();
  });
});
