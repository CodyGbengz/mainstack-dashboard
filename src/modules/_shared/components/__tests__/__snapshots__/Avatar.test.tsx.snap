// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`COMPONENT TEST: Avatar > Should match snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-bdfCkl iCZpDR"
    >
      <img
        alt="user avatar"
        src="/src/assets/userImage.png"
      />
    </div>
  </div>
</body>
`;
